src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const lines = sourceString
+    .split(';')
+    .map((line) => line.trim())
+    .filter((line) => line.length > 0);
+
+  lines.forEach((line) => {
+    const [property, value] = line.split(':').map((part) => part.trim());
```

Comment:
The `split(':')` operation assumes that each line contains exactly one colon. Consider handling cases where there might be more than one colon or none at all to avoid unexpected behavior.